inaugurator/ceremony.py

```diff
@@ -313,20 +314,43 @@ def _additionalDownload(self, destination):
             downloadInstance.download(destination)
 
     def _makeSureDiskIsMountable(self):
-        if self._args.inauguratorTargetDeviceCandidate is None:
+        self._setTargetDevice()
+        self._createPartitionTable()
+        logging.info("Partitions created")
+        self._mountOp = mount.Mount()
+        assert self._bootPartitionPath is not None, "Please initialize boot partition path first"
+        self._mountOp.setBootPartitionPath(self._bootPartitionPath)
+
+    def _setTargetDevice(self):
+        if self._args.inauguratorTargetDeviceCandidate is not None:
+            logging.info("Explicit target devices indicated: %(deviceCandidates)s",
+                         dict(deviceCandidates=self._args.inauguratorTargetDeviceCandidate))
+            candidates = self._args.inauguratorTargetDeviceCandidate
+        elif self._args.inauguratorTargetDeviceType is not None:
             logging.info("Searching for target devices of type %(deviceType)s",
                          dict(deviceType=self._args.inauguratorTargetDeviceType))
             device = self._storageDevices.findFirstDeviceOfType(self._args.inauguratorTargetDeviceType,
```

Comment:
why first? and not every device - and then assert? I think it would be safer, no?